cmd/main.go

```diff
@@ -163,6 +176,12 @@ func main() {
 		messengers: make(map[string]messenger.Messenger),
 		log:        lo,
 		bufLog:     bufLog,
+		metrics: metrics.MetricsConfig{
+			Prometheus: metrics.PrometheusConfig{
```

Comment:
You can remove the nested `Prometheus` struct here. Make `Enabled` and `Path` as top level fields to `MetricsConfig`.